src/Http/Http.Abstractions/src/Resources.resx

```diff
@@ -162,4 +162,13 @@
   <data name="RouteValueDictionary_DuplicatePropertyName" xml:space="preserve">
     <value>The type '{0}' defines properties '{1}' and '{2}' which differ only by casing. This is not supported by {3} which uses case-insensitive comparisons.</value>
   </data>
+  <data name="Exception_KeyedServicesNotSupported" xml:space="preserve">
+    <value>This service provider doesn't support keyed services.</value>
+  </data>
+  <data name="Exception_NoServiceRegistered" xml:space="preserve">
+    <value>No service for type '{0}' has been registered.</value>
+  </data>
+  <data name="Exception_ActivateMiddlewareNoService" xml:space="preserve">
```

Comment:
We shouldn't add resource strings that are only used by tests.

Reply:
Ah, agreed, I'll remove it from the resource. Shall I add it as a constant to the testmethod?

Reply:
Inline is fine

Reply:
I've pushed the changes 👍